humanmade/asset-manager-framework
# Fix crop with local provider

### Testing steps: 
1. Enable local media `add_filter( 'amf/allow_local_media', '__return_true' );`
1. Open customizer -> Site identity -> Logo. 
1. Open image, Crop. It should not throw JS error and crop the specified area as a new attachment.

<details><summary>Screencast before</summary>
<p>

https://user-images.githubusercontent.com/5015489/126511521-acf4cfa8-fceb-4fa8-b13a-dc008effdcea.mp4

</p>
</details>

<details><summary>Screencast after</summary>
<p>

https://user-images.githubusercontent.com/5015489/126511623-2a6c2d1a-7643-4622-8435-1c7230da6739.mp4


</p>
</details>
